doc/source/skips/1-governance.rst

```diff
@@ -0,0 +1,165 @@
+members of the community. All non-sensitive project management discussion takes
+place on the project `mailing list <mailto:scikit-image@python.org>`_
+and the `issue tracker <https://github.com/scikit-image/scikit-image/issues>`_.
+Occasionally, sensitive discussion may occur on a private list.
+
+Decisions should be made in accordance with the :ref:`mission, vision and
+values <values>` of the scikit-image project.
+
+Scikit-image uses a “consensus seeking” process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+If no option can be found without objections, the decision is escalated to the
+SC, which will itself use consensus seeking to come to a resolution. In the
+unlikely event that there is still a deadlock, the proposal will move forward
+if it has the support of a simple majority of the SC. Any vote must be
+backed by a :ref:`scikit-image proposal (SKIP) <skip>`.
+
+Decisions (in addition to adding core developers and SC membership as above)
+are made according to the following rules:
+
+- **Minor documentation changes**, such as typo fixes, or addition / correction of a
+  sentence (but no change of the scikit-image.org landing page or the “about”
+  page), require approval by a core developer *and* no disagreement or requested
+  changes by a core developer on the issue or pull request page (lazy
+  consensus). Core developers are expected to give “reasonable time” to others
+  to give their opinion on the pull request if they’re not confident others
+  would agree.
+
+- **Code changes and major documentation changes** require agreement by *two*
+  core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull-request page (lazy consensus).
+
+- **Changes to the API principles and changes to dependencies or supported
+  Python versions** require a :ref:`skip` and follow the decision-making
+  process outlined above.
+
+- **Changes to this governance model or our mission, vision, and values**
+  require a :ref:`skip` and follow the decision-making process outlined above.
+
+If a veto is cast on a lazy consensus, the proposer can appeal to the
+community and core developers and the change can be approved or rejected using
+the decision making procedure outlined above.
+
+.. _skip:
+
+Improvement proposals (SKIPs)
+=============================
+For all votes, a formal proposal must have been made public and discussed before the
+vote. The lifetime of a SKIP is as follows:
+
+- A proposal is brought up as either a GitHub issue or a post to the mailing
+  list.
+- After sufficient discussion, the core advocate(s) of the proposal must consolidate
+  the discussion into a single document, with appropriate references to the
+  original discussion, representing the pros and cons brought up by each
+  participant.
+- Such proposal must be a consolidated document, in the form of a
```

Comment:
Does this point repeat the previous?

Reply:
A little bit. I've consolidated (heh) the two bullet points.